frontend/packages/helm-plugin/console-extensions.json

```
@@ -97,6 +97,14 @@
       "required": ["OPENSHIFT_HELM"]
     }
   },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/ns/:ns/project-helm-chart-repository/~new"],
```

should this be plural?
```suggestion
      "path": ["/ns/:ns/projecthelmchartrepositories/~new"],
```